src/jsdoc_transformer.ts

```diff
@@ -893,8 +902,9 @@ export function jsdocTransformer(
           }
           exportDecl = ts.updateExportDeclaration(
               exportDecl, exportDecl.decorators, exportDecl.modifiers,
-              ts.createNamedExports(exportSpecifiers), exportDecl.moduleSpecifier);
-        } else {
+              ts.createNamedExports(exportSpecifiers), exportDecl.moduleSpecifier, false);
```

Comment:
Can you give the `false` a name?

```suggestion
          const withBananas = false;
          exportDecl = ts.updateExportDeclaration(
              exportDecl, exportDecl.decorators, exportDecl.modifiers,	              exportDecl, exportDecl.decorators, exportDecl.modifiers,
              ts.createNamedExports(exportSpecifiers), exportDecl.moduleSpecifier);	              ts.createNamedExports(exportSpecifiers), exportDecl.moduleSpecifier, withBananas);
```

Reply:
Added as `isTypeOnlyExport`.